main.sh

```diff
@@ -29,15 +22,25 @@ sudo ninja -C build install
 sudo find /usr/local/lib -name 'librte_*.a' -delete
 sudo ldconfig
 
-cd $CODEROOT/spdk_$SPDKVER
-if ! [[ -f build/lib/libspdk_env_dpdk.a ]]; then
-  ./configure --target-arch=$TARGETARCH --enable-debug --disable-tests --with-shared \
-    --with-dpdk=/usr/local --without-vhost --without-isal --without-fuse
-  make -j$(nproc)
+if [[ ! -z $SPDKVER && -n $SPDKVER && $SPDKVER != "none" ]]; then
```

Comment:
Keep `! && ||` operators outside of `[[ ]]` brackets. Use multiple `[[ ]]` brackets. This makes the condition more readable.